packages/terra-list/src/ListItem.scss

```diff
@@ -1,30 +1,23 @@
 @import '~terra-mixins';
 @import './variables';
 
-.terra-ListItem-hasChevron {
-  display: block;
-  position: relative; // relative position so the chevron can use absolute position to vertically center itself
+.terra-ListItem {
+  display: table;
+  outline: none;
```

Comment:
Remove `outline: none;` @neilpfeiffer mentioned we want to maintain the native focus outline style.